adapters/zeta_global_ssp/zeta_global_ssp.go

```diff
@@ -36,15 +54,37 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 
 	requestData := &adapters.RequestData{
 		Method:  "POST",
-		Uri:     a.endpoint,
-		Body:    requestJson,
+		Uri:     endpointURL,
+		Body:    requestJSON,
 		Headers: headers,
 		ImpIDs:  openrtb_ext.GetImpIDs(request.Imp),
 	}
 
 	return []*adapters.RequestData{requestData}, nil
 }
 
+func (a *adapter) getImpressionExt(imp *openrtb2.Imp) (*openrtb_ext.ImpExtZetaGlobalSsp, error) {
+	var bidderExt adapters.ExtImpBidder
+	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
+		return nil, &errortypes.BadInput{
+			Message: "ext.bidder not provided",
+		}
+	}
+	var zetaSspImpExt openrtb_ext.ImpExtZetaGlobalSsp
+	if err := json.Unmarshal(bidderExt.Bidder, &zetaSspImpExt); err != nil {
+		return nil, &errortypes.BadInput{
+			Message: "ext.bidder not provided",
+		}
```

Comment:
Please add a supplemental JSON test that forces this error condition. Simply setting `imp.ext.bidder` to an empty string should suffice.